### DISCUSSION TITLE
Cheerio's New Website

### DISCUSSION THREAD

**Discussion by fb55** [cheeriojs/cheerio#3002] [General]
The new website is live (https://cheerio.js.org/ — see https://github.com/cheeriojs/cheerio/pull/2950)! This discussion is currently linked at the top of the website. Let's use this thread for feedback, and to track things that are still missing.

Known issues:

- [x] Sponsors are currently not shown on the website.
- [x] Algolia search doesn't work
- [x] Some API docs don't contain the actual content

**Comment by arndom** [comment]
Looks good 👌 I'm new to cheerio and having a demo on the site would help in deciding if this is the right package for you

**Reply by fb55** [reply]
That's great feedback. Do you have an example for a library that does this well?

**Reply by arndom** [reply]
[Editor.js](https://editorjs.io/) and [Draft.js](https://draftjs.org/) have some good demos you could check out

**Comment by usamajamil43** [comment]
Hi, 
I was going through the website and found the typo [there](https://cheerio.js.org/docs/basics/loading). 

"The load method is the most basic way to parse an HTML or XML document with Cheerio. It takes a string containing the document as its argument and returns a Cheerio object that you can use to traverse and manipulate the document.

Here's an example of how to use the load method:

```
import * as cheerio from 'cheerio';

const $ = cheerio.load('<html><body><h1>Hello, world!</h1></body></html>');

console.log($('title').text());      # It won't print anything because there is no title in the document string, but you have mentioned that 
                                                        it will print "Hello world!"
// Output: Hello, world!
```

"

**Comment by Roosteridk** [comment]
Algolia search doesn't return any results at the moment.

**Reply by fb55** [reply]
I'm in contact with Algolia trying to fix it 👍

**Reply by fb55** [reply]
Fixed this!

**Comment by edemaine** [comment]
I notice that some doc strings aren't showing up in the API docs. For example: https://cheerio.js.org/docs/api/classes/Cheerio#next has no explanation of what the method does, and perhaps related, the link to the source code is to [the wrong line](https://github.com/cheeriojs/cheerio/blob/f20ef5f/src/api/traversing.ts#L120). On the other hand, [the correct line](https://github.com/cheeriojs/cheerio/blob/f20ef5f/src/api/traversing.ts#L380) has a nice doc string above it, which would be great to include in the API docs.

**Reply by fb55** [reply]
Enabling `isolatedDeclarations` fixed this.

**Comment by michaelfaith** [comment]
Really like what this library brings to the table.  It was super easy to use, and the website was helpful in getting up and running.  My only suggestion is to add an example to the "Modifying Element Attributes and Properties" section for removing attributes (or a new section dedicated to it, like there is for removing elements).  

Any thoughts on when the 1.0 release will go live?  (There are good updates that have merged, but not yet released.)
